subxt/src/config/default_extrinsic_params.rs

```diff
@@ -16,8 +16,7 @@ pub type DefaultExtrinsicParams<T> = signed_extensions::AnyOf<
         signed_extensions::CheckGenesis<T>,
         signed_extensions::CheckMortality<T>,
         signed_extensions::ChargeAssetTxPayment<T>,
-        signed_extensions::SkipCheckIfFeeless,
-        signed_extensions::ChargeTransactionPayment,
+        signed_extensions::SkipCheckIfFeeless<T, signed_extensions::ChargeTransactionPayment>,
```

Comment:
This will break polkadot, westend and rococo.

AFAIU, only substrate-node has this so far

Reply:
https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/westend/src/lib.rs#L836

Reply:
AnyOf is clever, and will just ignore signed extensions that aren't used on a given node, so it's always good to add all of the common ones to this list (other comment notwithstanding) :)

Reply:
Ooh and yeah, let's not remove the charge asset one from the list, which I just noticed and maybe that ls what you were saying on second read :)

Reply:
All right, thanks.

So you are saying that we shall have both `SkipCheckIfFeeLess` and `ChargeTransactionPayment` here, correct?